Formula/a/asdf.rb

```diff
@@ -1,8 +1,7 @@
 class Asdf < Formula
   desc "Extendable version manager with support for Ruby, Node.js, Erlang & more"
   homepage "https://asdf-vm.com/"
-  url "https://github.com/asdf-vm/asdf/archive/refs/tags/v0.15.0.tar.gz"
-  sha256 "d0cafe61d27b5e3fcb53658821bfbf744fd040a8ea28b0e22277e032b8e8f7fe"
+  url "https://github.com/asdf-vm/asdf.git", revision: "c4e55259bb5df08aaf3486dc1cf80457498f551e", tag: "v0.16.0"
```

Comment:
can you maybe use tarball?

Reply:
Yes, I could switch to tarball if that is preferred.

Reply:
```suggestion
  url "https://github.com/asdf-vm/asdf/archive/refs/tags/v0.16.0.tar.gz"
  sha256 "e2df22d3943911eec5b2d095d8a89a55cc7da912095d3f806619fd597201b124"
```

Yes, we only check out a tag/revision if the formula can't work with a tag tarball. I've been cleaning up existing tag/revision usage in formulae recently, so please do continue to use a tag tarball here.

Reply:
PR updated. I've switched to downloading the `tar.gz` file instead.